data_filter_azure/data_filter_azure/containerregistry_server.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+import os
+import requests
+import base64
+import json
+from flask import Flask,redirect, jsonify, abort, make_response, g, Response
+import config
+from flask_bootstrap import Bootstrap
+import subprocess
+import azure.common
+from data_filter_azure import opa
+import azure.cosmos.cosmos_client as cosmos_client
+
+app = Flask(__name__)
+Bootstrap(app)
+
+def check_access_opa(registry_id, user_id, resource_type, resource_name, action):
+    decision = opa_get(registry_id, user_id, resource_type, resource_name, action)
+    return decision
+
```

Comment:
I don't see why this is necessary. Authorization requests should still go to OPA, not to this bundle server, right?  OPA uses this bundle server to download bundles, but authorization requests still go to OPA.  Or at least, that's what I'd recommend because then you could have 5 OPA instances all pulling bundles from 1 bundle server.